elastic/kibana
# [Fleet] Fix uninstall package validation accross space

## Summary

Resolve https://github.com/elastic/kibana/issues/187632

That PR fix the logic to disable the uninstall button to work with the space awareness initiative.

Move the logic to disable the uninstall package button server side has the client may not have the permissions to query package policies across all spaces.

```
GET /.../nginx/{version}?withMetadata=true
{
 "item": {},
 "metadata": {
    "has_policies": true
  }
}
```

Fix the server side validation to query package policies across all spaces.

## How to test

Create multiple space, add a package policy and agents only in one, and try to uninstall a used package from a space where there is no package policy or agents => the UI should be disabled and the API should reject the request.









Work in progress
